### PR TITLE
feat(apps): bunch of new apps

### PR DESCRIPTION
**Pull request by H4M5TER**
- AutoHotkeyUX
- [Clash Verge](https://github.com/zzzgydi/clash-verge)
- [CopyQ](https://hluk.github.io/CopyQ/)
- [Everything](https://www.voidtools.com/)
- [Golden Dict](http://goldendict.org/)
- ~[ManicTime](https://www.manictime.com/)~
- [OneQuick](https://onequick.org/)
- QQ
- [Sandboxie Plus](https://sandboxie-plus.com/)
- [NetEase Cloud Music](https://music.163.com/)
---
- [x] I have formatted `applications.yaml` with `komorebic fmt-asc`.

### CODE REVIEW

**Review by sitiom — CHANGES_REQUESTED** [review]
I have applied some of my suggestions. Note that we only float fixed-size windows (which won't get appropriately managed with the tiling layout), so I have removed those unnecessary rules. Please see the rest of the comments.

File: applications.yaml [hunk 1]
```diff
@@ -739,6 +807,9 @@
   - kind: Title
     id: notificationtoasts_
     comment: Target notification toast popups
+  - kind: Title
+    id: 好友列表
+    comment: Friend list in Chinese
```
Comment (sitiom) [inline applications.yaml:812]:
```suggestion
```

**Review by sitiom — CHANGES_REQUESTED** [review]

File: applications.yaml [hunk 2]
```diff
@@ -507,6 +550,12 @@
     id: obs64.exe
   options:
   - tray_and_multi_window
+  float_identifiers:
+  - kind: Title
+    id: 设置
+  - kind: Class
+    id: Qt643QWindowPopupDropShadowSaveBits
+    comment: Menu bar popup
```
[the remark this answers is not in the document]
Comment (sitiom) [reply applications.yaml:546]:
```suggestion
```

**Review by sitiom — APPROVED** [review]

**Comment by H4M5TER** [comment]
Thank you for your review
Let me explain the questions
1. Floating rules. The others is personal but for CopyQ it's a clipboard software when intented to pop up and disappear again. I'm sure most people don't want its position to be change by komorebi.
2. ManicTime. Is there a way to make ManicTime managed by komorebi?
3. OBS. 
3.1 Menu bar pop up is important. When you click the menu bar the pop up will appear (red rectangle). It will disrupt the layout without the floating rule.
image
3.2 `设置` actually means `Settings`. According to your explanation, OBS's settings is not fixed sized so just delete it.

**Comment by H4M5TER** [comment]
I have made #43

**Comment by sitiom** [comment]
> 1. Floating rules. The others is personal but for CopyQ it's a clipboard software which intended to pop up and disappear again. I'm sure most people don't want its position to be change by komorebi.

I used to use CopyQ on Linux. The way I use it is that I toggle the tray popup (which is not managed by the tiling wm) instead of the main window. By default, that should cover most use cases. If you want to float the main window, you pass your own `overrides.yaml` containing the rule.

> 2\. ManicTime. Is there a way to make ManicTime managed by komorebi?

Upon testing, yes: https://github.com/LGUG2Z/komorebi-application-specific-configuration/commit/88bbbe25130bdefa2a312762312f444ff5ceea05

> 3\. 3.1 Menu bar pop up is important. When you click the menu bar the pop up will appear (red rectangle). It will disrupt the layout without the floating rule.

It works just fine for me without the rule.

> 4\. Border overflow. Can you explain what the border overflow option means for me? I'm not sure about that.

Some apps (most notably Electron apps) have borders that overflow beyond the layout area and cover the gap space around it. Here's an example:

Without the rule:
image
With the rule:
image

> 5\. Everything. I have two screen which is zoomed differently. When the Everything window is on the part of main screen which is next to another screen, it will be zoomed incorrectly. With border overflow option it works fine.

Can you elaborate more? Do you mean when Everything is between the two screens?

**Comment by H4M5TER** [comment]
good case when window on left
c817fc03c0dd680d763de69c4650583d

good case when window on right with border overflow option
3a69298a12e5d87123826b91a1cb32c7

bad case when window on right without option
45b5534e5db7816daac7130fa32c3689

**Comment by H4M5TER** [comment]
You are right. I can't reproduce the OBS issue. 
Can you explain more about how the `overrides.yml` works?

**Comment by sitiom** [comment]
Window extending from the first screen to the second screen? I also have two screens with different zoom sizes, and I can't seem to replicate this. @LGUG2Z A bug from komorebi perhaps?

> You are right. I can't reproduce the OBS issue.
> Can you explain more about how the `overrides.yml` works?

If you are using a static configuration, you can define your own rules directly in the `komorebi.json`

image

Alternatively, if you use an AutoHotkey config, you can run the `ahk-app-specific-configuration` or `ahk-asc` command to generate the script for the rules:
```
komorebic ahk-asc applications.yaml overrides.yaml
```

**Pull request merged**